test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml

```diff
@@ -1406,6 +1406,12 @@
 		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
 	</test>
 
+	<test id="Test 66: Processor Features PrintStats Test : Ensure -Xshareclasses:printStats cache contains processor feature info" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$,printStats -version</command>
+		<output type="failure" caseSensitive="yes" regex="yes" javaUtilPattern="yes" showMatch="yes">Processor Features[\s]*= null</output>
+		<output type="success" caseSensitive="yes" regex="yes" javaUtilPattern="yes" showMatch="yes">Processor Features[\s]*= .*</output>
```

Comment:
These tests are only going to work on x86 platforms, they need to be restricted to these platforms. There is a way to do this, look for an example in the `ShareClassesCMLTests-*.xml` files

Reply:
@pshipton Ready for review.